### PR TITLE
Add common package

### PR DESCRIPTION
**Pull request by RustySol**
Common package should contain all the API/program
call related logic, to be able to support cli in
the coming changes.

### CODE REVIEW

File: common/src/api/backend/OrderAPI.ts [hunk 1]
```diff
@@ -14,9 +14,10 @@ export type OrdersFilterQuery = {
 };
 
 export async function fetchOrdersByStoreId(
+  axiosInstance: AxiosInstance,
   storeId: string,
   ordersFilterQuery: OrdersFilterQuery,
-  identifiers?: number[]
+  identifiers?: number[],
```
Comment (haVincy) [inline common/src/api/backend/OrderAPI.ts:20]: nit - additional comma?

File: common/src/api/backend/NftAPI.ts [hunk 2]
```diff
@@ -1,7 +1,10 @@
 import { SingleBase, Nft } from 'solana-candy-shop-schema/dist';
-import axiosInstance from '../../config/axiosInstance';
+import { AxiosInstance } from 'axios';
```
Comment (haVincy) [inline common/src/api/backend/NftAPI.ts:2]: What's difference between previous axio usage and current one, seems this one is from `axio` library itself?

File: lib/src/core/Sell/index.tsx [hunk 3]
```diff
@@ -14,7 +12,11 @@ import {
   Order as OrderSchema,
   WhitelistNft,
 } from 'solana-candy-shop-schema/dist';
-import { CandyShop } from 'core/CandyShop';
+import {
+  CandyShop,
+  fetchNftsFromWallet,
+  SingleTokenInfo,
```
Comment (haVincy) [inline lib/src/core/Sell/index.tsx:18]: nit - additional comma?

File: lib/src/index.tsx [hunk 4]
```diff
@@ -1,9 +1,8 @@
 import './index.less';
 
-export { CandyShop, CandyShopSettings } from './core/CandyShop';
+export * from '@liqnft/candy-shop-common';
```
Comment (haVincy) [inline lib/src/index.tsx:3]: I think we should specify the import here? Any reason that must using `*`?

File: common/src/api/backend/OrderAPI.ts (hunk 1 above)
Comment (RustySol) [reply common/src/api/backend/OrderAPI.ts:20]: this is from the formatter, somehow wasnt formatted before

File: common/src/api/backend/NftAPI.ts (hunk 2 above)
Comment (RustySol) [reply common/src/api/backend/NftAPI.ts:2]: there is no difference, adding this is just for importing the typing
and make sure when the candyShop calls these methods, the axiosInstance is passed in from candyShop class

File: common/src/api/backend/OrderAPI.ts (hunk 1 above)
Comment (haVincy) [reply common/src/api/backend/OrderAPI.ts:20]: kk, I think I know why, the prettierrc should add ` trailingComma: 'none'`, I will take care this in my PR.

**Review by haVincy — APPROVED** [review]
LGTM.

**Pull request merged**